editor/plugins/script_editor_plugin.cpp

```diff
@@ -1584,25 +1584,27 @@ void ScriptEditor::_update_script_names() {
 	}
 
 	if (_sort_list_on_update) {
-		sedata.sort();
-
-		// change actual order of tab_container so that the order can be rearranged by user
-		int cur_tab = tab_container->get_current_tab();
-		int prev_tab = tab_container->get_previous_tab();
-		int new_cur_tab = -1;
-		int new_prev_tab = -1;
-		for (int i = 0; i < sedata.size(); i++) {
-			tab_container->move_child(sedata[i].ref, i);
-			if (new_prev_tab == -1 && sedata[i].index == prev_tab) {
-				new_prev_tab = i;
-			}
-			if (new_cur_tab == -1 && sedata[i].index == cur_tab) {
-				new_cur_tab = i;
+		if (!sedata.empty()) {
```

Comment:
This could be an extra condition in the previous `if` instead of another nested `if` block.

Reply:
it's updated now